components/authz-service/engine/conformance/conformance_v2_test.go

```diff
@@ -362,7 +359,7 @@ func TestV2ProjectsAuthorized(t *testing.T) {
 			})
 
 			t.Run("policy that allows all projects returns all when all projects requested", func(t *testing.T) {
-				proj3, proj4 := "proj-3", "proj-4"
+				proj3, proj4, unassigned := "proj-3", "proj-4", "(unassigned)"
```

Comment:
This line is a phantom; delete it please.

Reply:
nice catch!